OutlookMatters.Core.Test/Http/HttpClientMockRestExtensions.cs

```diff
@@ -55,15 +55,16 @@ public static Mock<IHttpResponse> WithToken(this Mock<IHttpResponse> httpRespons
         public static Mock<IHttpResponse> FailsAtGet(this Mock<IHttpRequest> httpRequest)
         {
             var httpResponse = new Mock<IHttpResponse>();
-            var httpException = new HttpException(httpResponse.Object);
+            var httpException = new ServiceException(httpResponse.Object);
             httpRequest.Setup(x => x.Get()).Throws(httpException);
             return httpResponse;
         }
 
         public static Mock<IHttpResponse> FailsAtPost(this Mock<IHttpRequest> httpRequest, string payload)
         {
             var httpResponse = new Mock<IHttpResponse>();
-            var httpException = new HttpException(httpResponse.Object);
+            var httpException = new ServiceException(httpResponse.Object);
+            httpRequest.Setup(x => x.WithContentType("text/json")).Returns(httpRequest.Object);
```

Comment:
Just a note, Mattermost Api v4 expexts/provides request and response-bodies as content-type `application/json`

Reply:
Thanks, you're right. I removed that not necessary statement because of the **WithContentType** extension at line 37.